lca-cli/ops/execute.go

```diff
@@ -57,6 +65,10 @@ func NewRegularExecutor(logger *logrus.Logger, verbose bool) Execute {
 }
 
 func (e *regularExecutor) Execute(command string, args ...string) (string, error) {
+	if stringslices.Contains(args, "|") {
+		// The command includes a pipe, so run it with bash -c
+		e.executor.executeBash(nil, "", command, args...)
```

Comment:
This makes me very uneasy from a security perspective. Nothing concrete but shells are injection paradise. If we need a shell, we better be very explicit about it in the calling function

Reply:
Removing